openupgrade_scripts/scripts/hr/16.0.1.1/post-migration.py

```diff
@@ -76,6 +76,9 @@ def create_work_contact(env):
                     employee.id,
                 )
             else:
+                if not employee.work_email and not employee.mobile_phone:
+                    # If you do not have any data, it is not necessary to create a contact.
+                    continue
```

Comment:
Sorry for not being very clear. I was talking about:

```suggestion
            elif employee.work_email or employee.mobile_phone:
```